### ISSUE TITLE
Request: Aerox 5 Wireless

### ISSUE THREAD

**Issue by aknotek** [flozz/rivalcfg#184]
Device identification wireless: 1038:1852
Device identification wired mode: 1038:1854

Product URL: https://steelseries.com/gaming-mice/aerox-5-wireless

Hey there! I have an Aerox 5 wireless. I was able to get it working using the existing Aerox 3 wireless code, just changing the device ID in aerox3_wireless_wireless.py. All the features for for Aerox 3 appear to work on the Aerox 5. Please let me know if there's any testing you need me to do!

**Comment by flozz** [comment]
Hello,

There is a difference between the Aerox 3 and 5: the buttons. So button mapping may not work correctly.

---

I started working on this device on the `aerox-5-wireless` branch. Can you check everything is working in both wireless and wired mode:

* [ ] `--sensitivity`
* [ ] `--polling-rate`
* [ ] `--top-color`
* [ ] `--middle-color`
* [ ] `--bottom-color`
* [ ] `--reactive-color`
* [ ] `--sleep-timer`
* [ ] `--dim-timer`
* [ ] `--rainbow-effect`
* [ ] `--battery-level` (if possible test with battery fully charged and with battery low to see if there is no negative number or level > 100%)

---

For buttons, I will try to find some more information and I add hare a screenshot I found on the web that show the buttons in the GG engine (for later reference):

![image](https://user-images.githubusercontent.com/971438/185910353-d8f49222-0d61-4b9e-8904-a3dbb9194dae.png)

**Comment by flozz** [comment]
I implemented button mapping:

```
  -b BUTTONS_MAPPING, --buttons BUTTONS_MAPPING
                        Set the mapping of the buttonsX (default: buttons(button1=button1;
                        button2=button2; button3=button3; button4=button4; button5=button5;
                        button6=dpi; button7=disabled; button8=disabled; button9=disabled;
                        scrollup=scrollup; scrolldown=scrolldown; layout=qwerty))
```

I need someone to check it works too :)

**Comment by aknotek** [comment]
Thanks for your work, Flozz! I will get these tested today, probably after dinner.

**Comment by aknotek** [comment]
I was able to test everything on the list in wired and wireless mode. 
--battery-level looks fine from 20%-100%. Can let it drop below 20% tomorrow and report back if needed.
--sleep-timer works fine on wireless mode but on wired the mouse doesn't appear to go to sleep. Maybe this is because it's plugged in?

Everything else on the list works as expected.

Will test the buttons either tomorrow on the next day let you know how it goes.

**Comment by flozz** [comment]
>  --sleep-timer works fine on wireless mode but on wired the mouse doesn't appear to go to sleep. Maybe this is because it's plugged in?

Yes, the sleep mode can be configured in both mode but is active only in wireless :)

**Comment by aknotek** [comment]
Alrighty, I was able to test the button remapping for all the buttons in wired and wireless modes. They all worked as they should.

**Comment by aknotek** [comment]
Was able to get the battery to report a negative number. It was about 80% then I saw this. `-5%`
![image](https://user-images.githubusercontent.com/28696376/186961002-59918931-2931-4f4e-8153-30d9193a1f78.png)

**Comment by flozz** [comment]
It goes from 80 to -5 instantly? oO'

**Comment by aknotek** [comment]
Yes, but I don't think rivalcfg is to blame. Messed with this a bit and I believe the wireless receiver was acting up as everything went back to normal once I unplugged and plugged it back in to the USB.

**Comment by flozz** [comment]
Ok, good to know :)

**Comment by flozz** [comment]
The support of this device has been merged to `master`, it will be released with next Rivalcfg version (v4.7.0). Thank you for your help! :)
